lib/OpenQA/Schema/Result/Users.pm

```diff
@@ -53,7 +53,7 @@ __PACKAGE__->add_columns(
     },
     feature_version => {
         data_type     => 'integer',
-        default_value => 1,
+        default_value => 0,
```

Comment:
I disabled the feature tour temporarly, since i dont have real tour yet. Will do a real tour in the next step, but i thought it would be a good idea to make a merge of the branch possible.